src/strings.cpp

```diff
@@ -2042,8 +2042,7 @@ static void GetLanguageList(const char *path)
 void InitializeLanguagePacks()
 {
 	for (Searchpath sp : _valid_searchpaths) {
-		std::string path = FioGetDirectory(sp, LANG_DIR);
-		GetLanguageList(path.c_str());
+		GetLanguageList(FioGetDirectory(sp, LANG_DIR));
```

Comment:
this function being named `Get` but returning nothing is... unusual. `Set` seems better, ironically.